### ISSUE TITLE
Feature Request: Autoplay Videos on Focus

### ISSUE THREAD

**Issue by DeadSuperHero** [pixelfed/mobile-app#66]
This is more of a usability papercut than anything else, but: it would be great if a video started autoplaying as soon as part of the video comes into the view focus. Having to tap to start or stop isn't a big deal, but autoplay could make that experience a little more seamless.

**Comment by twann4** [comment]
I don't think this is a good idea in terms of ethics. Autoplaying videos is one of the features that makes corporate social media so addictive. I can only hope that the fediverse won't go that way.
